hokusai/services/deployment.py

```diff
@@ -88,6 +93,7 @@ def update(self, tag, constraint, git_remote, timeout, update_config=False, file
           for container in item['spec']['template']['spec']['containers']:
             if self.ecr.project_repo in container['image']:
               container['image'] = "%s@%s" % (self.ecr.project_repo, digest)
+              container['env'] = add_env(container, {'name': "%s" % app_version_var_name, 'value': "%s" % digest})
```

Comment:
Is this `digest` the whole long thing starting in `sha256:77af4d6b...`?

I don't remember exactly what the Datadog UI does, but maybe we should truncate this or strip the leading `sha256:`. (Ideally we'd have something more readable like the timestamps, but I don't think we have a "permanent" timestamp that spans environments handy.)

Reply:
yeah stripping `sha256:` should be fine. timestamp could be useful, but wonder if it'd be confusing if the same image is deployed multiple times. i actually wanted to use git commit sha but refrained seeing how those tags in ECR are missing in git commit history (because of our squash-merge?)